src/test/java/com/rybka/service/connector/ExchangeRateConnectorTest.java

```diff
@@ -0,0 +1,45 @@
+package com.rybka.service.connector;
+
+import com.rybka.constant.ParameterValue;
+import com.rybka.model.ExchangeResponse;
+import coresearch.cvurl.io.request.CVurl;
+import coresearch.cvurl.io.request.RequestBuilder;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(SpringExtension.class)
+class ExchangeRateConnectorTest {
+
+    @Mock
+    private CVurl cVurl;
+    @Mock
+    private RequestBuilder requestBuilder;
+
+    private ExchangeRateConnector connector;
+
+    private ExchangeResponse response;
+
+    @BeforeEach
+    public void setUp() {
+        connector = new ExchangeRateConnector(cVurl);
+        response = new ExchangeResponse();
+        response.setBase(ParameterValue.USD);
+        response.setRates(Map.of(ParameterValue.EUR, ParameterValue.EUR_RATE));
+    }
+
```

Comment:
What about test of failures and exception handlings?

Reply:
fixed